### PR TITLE
Update intro to data commons notebook to fix covid 19 variable

### PR DESCRIPTION
**Pull request by juliawu**
Updates the COVID19 variable used in this notebook to `Count_MortalityEvent_COVID19`.  Other formatting changes are a result of re-exporting from colab, and colab's changes to default display settings.

Screenshot of last cell with the variable which got changed in red box:
image

You can also view a live version of the notebook being added here: https://colab.sandbox.google.com/github/juliawu/datacommons-api-python/blob/master/notebooks/intro_data_science/Data_Commons_For_Data_Science_Tutorial.ipynb

### CODE REVIEW

**Review by pradh — APPROVED** [review]
Hard to really review diffs, so blind LG :)

File: notebooks/intro_data_science/Data_Commons_For_Data_Science_Tutorial.ipynb [hunk 1]
```diff
@@ -3,10 +3,9 @@
   "nbformat_minor": 0,
   "metadata": {
     "colab": {
-      "name": "Data Commons For Data Science Tutorial.ipynb",
```
Comment (jehangiramjad) [inline notebooks/intro_data_science/Data_Commons_For_Data_Science_Tutorial.ipynb:6]: why the removal of the name?

File: notebooks/intro_data_science/Data_Commons_For_Data_Science_Tutorial.ipynb [hunk 2]
```diff
@@ -232,7 +231,7 @@
         "         \"geoId/065300\",   # Oakland\n",
         "         \"geoId/0644000\"]  # Los Angeles"
       ],
-      "execution_count": null,
+      "execution_count": 3,
```
Comment (jehangiramjad) [inline notebooks/intro_data_science/Data_Commons_For_Data_Science_Tutorial.ipynb:234]: Should the execution counts not be null? I guess it depends on whether we want the outputs to be visible by default or not?

**Review by jehangiramjad — APPROVED** [review]
Approving with a couple of questions. Why not also attach screenshots of the revised notebook so the byte encoded output can be seen in the review?

Thanks for the fix btw :)

File: notebooks/intro_data_science/Data_Commons_For_Data_Science_Tutorial.ipynb (hunk 1 above)
Comment (juliawu) [reply notebooks/intro_data_science/Data_Commons_For_Data_Science_Tutorial.ipynb:6]: This was not a change I made intentionally, it looks like colab's export metadata no longer includes name compared to 2 years ago.

**Pull request merged**